scripts/_aggregate_cells.py

```diff
@@ -0,0 +1,113 @@
+import os
```

Comment:
Please use [`black`](https://github.com/psf/black).

It literally changed my life! I never have to think about python formatting :)

Reply:
_aggregate_cells.py Removed on commit: https://github.com/WayScience/CytoPipe/commit/67828fb3af04336c529db4eebfd2c0f8f3958e04

Will do!